README.md

```diff
@@ -78,14 +78,13 @@ This will set up a new application on your Heroku account using this repo as a t
 
 -->
 
-## Deploy on CodeSandbox
+## Deploy on Deno Deploy
 
-1. Log in or register on [CodeSandbox - Sign in](https://codesandbox.io/signin)
```

Comment:
did we lose the "register" / "sign up" step? (not sure what word Deno Deploy ues)

Reply:
I would like to make it as simple as possible and students should use the "Continue with GitHub" button. Clarified the step in this commit [5b30ca3](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/366/commits/5b30ca34ae3ea19ba5ee4b9fffb03699c1ed9c4d)

<img width="1453" alt="Screenshot 2025-03-14 at 09 44 52" src="https://github.com/user-attachments/assets/d4b9243a-b3b4-4011-bd90-53b3d04f4d59" />

Reply:
does that appear when you're logged in already?

if not, then you should keep a separate "sign in" step

Reply:
No, it doesn't appear when you already logged in, and you are directly at the "Create new project" page. Added a new step that goes directly to the signin, and this opens the Github Signup
<img width="1464" alt="Screenshot 2025-03-14 at 16 50 30" src="https://github.com/user-attachments/assets/f1467e23-adfe-4628-83cc-4d3f9e837e2b" />

[6e07d1d](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/366/commits/6e07d1d5635ca9edc47a306c20012be8299dfc16)

Reply:
1. as you wrote me offline, this step has at least 1 extra step (maybe more), including an "authorization" step, so something in step 3 should also mention at least that authorization process - eg. using the word "authorize" somehow or even mentioning the button "Authorize Deno Deploy"
2. keep in mind that your words for step 3 should also mention the previous step (shown in your screenshot)
3. carefully go through this flow from the beginning (do this by revoking access to Deno Deploy from your account first), and check if there is a step that asks for access to specific repositories. if there is, then make sure that there is text to not select all repositories, but only the `express-guest-list-api-memory-data-store` repo

Reply:
Removed Deno from my account and created new screenshots. Added a new step for authorising Deno. I think the guide is clear now that students only need to allow access to the express guest list api repo.
[8bd9173](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/366/commits/8bd91734aca50e13b66d6bd0fca87e59a4bc1ca7)